tcms/static/js/testrun_actions.js

```diff
@@ -63,12 +64,22 @@ Nitrate.TestRuns.Details.on_load = function() {
     var case_id = c.find('input[name="case"]')[0].value;
     var case_run_id = c.find('input[name="case_run"]')[0].value;
     var case_text_version = c.find('input[name="case_text_version"]')[0].value;
-    var type = 'case_run';
     var callback = function(t) {
       // Observe the update case run stauts/comment form
+
+      const caseId = $('td > input[name=case_id]').val()
+      const executionId = $('td > input[name=object_pk]').val()
+
       c_container.parent().find('.update_form')
-        .unbind('submit').bind('submit', updateCaseRunStatus);
+        .unbind('submit').bind('submit', () => addComment(executionId, caseId));
+
+      const submitButtons = $('.submit_button')
```

Comment:
selector should search under `c`, not in the entire page.

If multiple rows are expanded there is the possibility for this to bind incorrect values to one of the buttons down the stack and issue RPC requests against the wrong TE.